case-lib/lib.sh

```diff
@@ -791,7 +791,7 @@ extract_format_number() {
     else
         die "Error: Unknown format: %s\n"
     fi
-}  
+}
```

Comment:
To keep git clean, don't mix pure whitespace changes with code changes; especially not when it's far away in a completely different file.
https://wiki.openstack.org/wiki/GitCommitMessages#Things_to_avoid_when_creating_commits